public/en/platform/protect-sdk-php/queue-documentation.md

```diff
@@ -0,0 +1,27 @@
+## Queue Documentation
```

Comment:
So just a heads-up, we have a new markdown linter for CH-19924 that isn't going to like this file. But it's an easy fix. Basically we just need to:

1. Replace ## with # on line 1 above
2. Replace ### with ## on lines 3 & 6
3. Wrap all lines so they are <= 80 characters

You created your PR first, so I'm happy to go in and make these changes if this PR gets merged before https://github.com/ns8inc/protect-sdk-php/pull/59. But if that PR goes in first then this is the reason why the Concourse tests will suddenly start failing.

Reply:
Good to know, thanks @davehensleyns8! I'll update it shortly.

Reply:
@davehensleyns8 Updated in 3de603bf301c96595dde1dadc9fa72581ae6e984.

Reply:
Nice! I just went ahead and merged #59 so hopefully the Concourse test will still succeed.

Reply:
And sure enough it did! 🎉

Reply:
in case it's helpful:  I use [this extension](https://marketplace.visualstudio.com/items?itemName=stkb.rewrap) for wrapping, and it does a pretty good job (i.e., it doesn't just wrap lines, but it's language/format aware and does the 'right thing')

Reply:
Thanks for the tip!